docs/minishift.md

```diff
@@ -10,15 +10,16 @@ Minishift is a CLI tool that provisions and manages single-node OpenShift cluste
 ### Options
 
 ```
-      --alsologtostderr                  log to standard error as well as files
-      --log-flush-frequency duration     Maximum number of seconds between log flushes (default 5s)
-      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
-      --log_dir string                   If non-empty, write log files in this directory (default "")
-      --logtostderr                      log to standard error instead of files
-      --show-libmachine-logs             Whether or not to show logs from libmachine.
-      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
-  -v, --v Level                          log level for V logs
-      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
+      --alsologtostderr value          log to standard error as well as files
+      --disable-update-notification    Whether to disable VM update check.
+      --log-flush-frequency duration   Maximum number of seconds between log flushes (default 5s)
+      --log_backtrace_at value         when logging hits line file:N, emit a stack trace (default :0)
+      --log_dir value                  If non-empty, write log files in this directory
+      --logtostderr value              log to standard error instead of files
+      --show-libmachine-logs           Whether or not to show logs from libmachine.
+      --stderrthreshold value          logs at or above this threshold go to stderr (default 2)
+  -v, --v value                        log level for V logs
```

Comment:
We can improve on this. I would like to have a closer look at this once we tackle the logging issue in itself. I don't want to change/document something I have not really investigated. 

BTW, the docs are generated. We need to change this in the Cobra command definition.